server/templates/partials/bottom/fried-frank-licence.html

```diff
@@ -0,0 +1,26 @@
+<div class="n-messaging-client-messaging-banner">
+	{{#> n-messaging-client/server/templates/components/o-banner
+		themeCompact=true
+		renderOpen=true
+		suppressCloseButton=true
+	}}
+	<div class="o-banner__inner" data-o-banner-inner="" data-o-banner-inner="">
+		<div class="o-banner__content">
+			<header class="o-banner__heading">
+				<h1>Fried, Frank, Harris, Shriver & Jacobson LLP Cancellation</h1>
+			</header>
+			<p>
+				Fried, Frank, Harris, Shriver & Jacobson LLP has decided not to renew their corporate subscription to FT.com. If you have any questions about this please email
+				<a href="mailto:FFHSJResearchServices@friedfrank.com">FFHSJResearchServices@friedfrank.com</a>.
+			</p>
+		</div>
+		<div class="o-banner__actions">
+			<div class="o-banner__action">
+				<a href="mailto:FFHSJResearchServices@friedfrank.com" class="o-banner__button">
```

Comment:
Why not automatically add a subject?
```suggestion
				<a href="mailto:FFHSJResearchServices@friedfrank.com?subject=Keep my access to FT.com" class="o-banner__button">
```
Not our call but maybe you can suggest it to UXD.